Wallet Download and Installation.md

```diff
@@ -0,0 +1,64 @@
+# Downloading and Installing Wallets to Your Devices
+
+Visit the **[Downloads](https://www.skycoin.net/downloads/)** page from the Skycoin Website to download the Skycoin Desktop Wallet and Skycoin Mobile Wallet for the device you are using:
+
+## [Desktop Wallet](Desktop_Wallet)
+
+* ### [Windows](Windows)
+* ### [MAC OS](MAC_OS)
+* ### [Linux](Linux)
+
+## [Mobile Wallet](Mobile_Wallet)
+
+* ### [IOS](IOS)
+* ### [Android](Android)
+
+## <a name="Desktop_Wallet"> </a> Desktop Wallet
+
+### <a name="Windows"> </a> Windows
+
+You can follow these steps to install the Skycoin Desktop Wallet into your Windows PC:
+
+* Clicking on the link, "64 bit" will start downloading the installer for the Wallet.
+
+* Once the installer is downloaded, double click to open it.
+
+* The installation of the Desktop Wallet is similar to the installation of any other Windows software and after confirming the actions, the installation will be completed.
```

Comment:
This is unnecessary context in my opinion. Please tell them to confirm the action and reference the button they have to  press.